package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@snyk/graphlib",
-  "version": "2.1.9-patch",
+  "name": "graphlib",
```

Comment:
?

Reply:
Tbh I was planning to include this as a git repo and ignore the release process / naming. But, yes, let's fix that, and use it in the CLI too, if we end up going for this?